CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# How to Contribute
+
+### Welcome
+
+First off, thank you for considering contributing to Opa Java Client. It's people like you that make this client such a great library.
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+### What we are looking for
+
+Opa Java Client is an open source project, and we love to receive contributions from our community — you! There are many ways to contribute, from writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests or writing code which can be incorporated into Opa Java Client itself.
+
+# Ground Rules
+
+Responsibilities
+* Ensure cross-platform compatibility for every change that's accepted. Windows, Mac, Debian & Ubuntu Linux.
+* Ensure that code that goes into core meets all requirements in this checklist.
+* Create issues for any major changes and enhancements that you wish to make. Discuss things transparently and get community feedback.
+* Don't add any classes to the codebase unless absolutely needed.
```

Comment:
I don't think this point is needed, doesn't really mean anything specific.

Reply:
yea, it's misleading